xbmc/platform/darwin/tvos/powermanagement/TVOSPowerSyscall.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (C) 2012-2018 Team Kodi
```

Comment:
Headers need to be changed. Maybe something like the below. Im not entirely sure on whats required, but the dates at least need to be current from when included.

Copyright (C) 2020 Team Kodi